tests/test_news_flash.py

```diff
@@ -212,11 +212,11 @@ def test_extract_location():
         assert getattr(actual, k) == getattr(expected, k)
 
 
-def test_extract_location_text():
+def test_location_extraction_extract_location_text():
```

Comment:
The second `location` is redundant (and longer names make it a tad more cumbersome to run a specific test).

Reply:
Because of a naming collision when trying to keep names short (`test_extract_location` is too generic) - I went with a name that is just a concatenation of the module and the function name that is being invoked
`location_extraction.extract_location_text` -> `test_location_extraction_extract_location_text`
As a convention this scales better than short vague names, while still not going with a deeper change of introducing modularization by classes, packages, or moving the tests near their designated packages (all good things but I tried to minimize infra changes in this short PR)
What would you prefer?

Reply:
I prefer without the second `location`, but your I understand your reasons. Resolving.